trades.py

```diff
@@ -670,10 +671,11 @@ def parse_user(  # pylint: disable=too-many-locals,too-many-statements
                 market_status
             ] += collateral_amount
             statistics_table[MarketAttribute.FEES][market_status] += fee_amount
+            mech_data = _mech_statistics.pop(fpmmTrade["title"], {})
             statistics_table[MarketAttribute.MECH_CALLS][
                 market_status
-            ] += mech_statistics.get(fpmmTrade["title"], {}).get("count", 0)
-            mech_fees = mech_statistics.get(fpmmTrade["title"], {}).get("fees", 0)
+            ] += mech_data.get("count", 0)
+            mech_fees = mech_data.get("fees", 0)
```

Comment:
If we do not pop, it will pick again the same values if there is another trade for the same question.  This is an issue, because there is no relation between mech calls and trades (we don't know what mech calls are related to each trade). Hence this was the solution adopted.